test/mock-server/deps.ts

```diff
@@ -0,0 +1,18 @@
+export type { Header } from 'https://deno.land/x/djwt@v1.9/mod.ts'
+
+export { create, getNumericDate } from 'https://deno.land/x/djwt@v1.9/mod.ts'
```

Comment:
sorry for the stupid question, but why fetching libraries like this?

Reply:
Oh because the mock server uses [Deno](https://deno.land/), a new JS runtime (just like Node), but it resolves dependencies differently